phoenix/slackbot/models.py

```diff
@@ -38,19 +36,14 @@ def save(self, force_insert=False, force_update=False, using=None,
     def dedicated_channel_name(self):
         date = self.outage.created.date().strftime('%y%m%d')
         systems = ','.join([system.name for system in self.outage.systems_affected.all()])[:9]
-        #offset = Outage.objects.filter(  # Ignore PEP8Bear
-            #created__day=self.outage.created.date().day,  # Ignore PyCommentedCodeBear
-            #pk__lt=self.outage.pk,
-            #systems_affected__in=self.outage.systems_affected.all(),
-        #).count()
+        offset = Outage.objects.filter(  # Ignore PEP8Bear
+            created__day=self.outage.created.date().day,  # Ignore PyCommentedCodeBear
```

Comment:
You can remove `# Ignore PyCommentedCodeBear` (and never add it again, what's the point, man :D)

Reply:
This was needed to be temporary commented out some time ago. In order to pass CI checks this were needed. I just forgot to remove them, going to do it now.

Reply:
yes, but there's a reason it was failing CI like that :D 

the point is that we use git for version control, so if you want to make temporary changes, you can just delete the code anyway and revert the commit where you made the changes

Reply:
Sure will do that next time.